src/PageControls.tsx

```diff
@@ -230,7 +230,18 @@ PageControls[Pages.digitalTwinAnalysis] = newInfoDialogControl({
 PageControls[Pages.winScreen] = newInfoDialogControl({
 	title: 'CONGRATULATIONS!',
 	text: (state) => `You succeeded at the goal. You managed to decarbonize {${state.companyName}} by {${(state.trackedStats.carbonSavings * 100).toFixed(1)}%} in 10 years!`,
-	img: 'images/confetti.png'
+	img: 'images/confetti.png',
+	buttons: [
+		{
+			text: 'Play again',
+			variant: 'text',
+			onClick: (state) => {
+				location.href = String(location.href); // Reload the page
```

Comment:
For now, since we are trying to move fast and intentionally just resetting everything this is fine but.... 
I'd be wary of doing something like this hard reset and dumping all user state. I'm not sure what the implications would be in production or if the app was managing routing. Just FYI